### PR TITLE
Automated cherry pick of #47: Bump go-build to pick up release process fixes

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #47 on release-v1.1.

#47: Bump go-build to pick up release process fixes

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**
